kotlin/integrated-windows-authentication/README.md

```diff
@@ -1,32 +1,47 @@
-# Integrated Windows Authentication
+# Integrated windows authentication
```

Comment:
Should windows be capitalized here? Idk.

Reply:
hmmmm. The Microsoft style guide says to capitalize product names "to help people buy them". I'll do it, but under protest.